### DISCUSSION TITLE
error to start SOC web console

### DISCUSSION THREAD

**Discussion by totoroha** [Security-Onion-Solutions/securityonion#11046] [General]
Dear Team,

I followed the setup here https://docs.securityonion.net/en/2.3/getting-started.html for installing SO on one of my VMware from ESXi system.
The VM comes with 4 CPU, 16 GB of RAM, and 200GB of disk space.
The login is as attachment
![SO error login](https://github.com/Security-Onion-Solutions/securityonion/assets/7328140/ff2c20c9-2cd8-4ced-9405-baa1531ce143)

I used sudo salt-call state.highstate command and got the following errors:

----------
          ID: xwindows_group
    Function: pkg.group_installed
        Name: X Window System
      Result: False
     Comment: An error was encountered while installing/updating group 'X Window System': Group 'X Window System' not found.
     Started: 08:08:58.108742
    Duration: 163.797 ms
     Changes:
----------
          ID: graphical_extras
    Function: pkg.installed
      Result: False
     Comment: Error occurred installing package(s). Additional info follows:

              errors:
                  - Running scope as unit run-29526.scope.
                    Loaded plugins: versionlock
                    No package liberation-mono-fonts available.
                    Error: Nothing to do
     Started: 08:08:58.273089
    Duration: 1441.242 ms
     Changes:
----------
          ID: graphical_target
    Function: file.symlink
        Name: /etc/systemd/system/default.target
      Result: False
     Comment: One or more requisite failed: workstation.packages.xwindows_group, workstation.packages.graphical_extras
     Started: 08:08:59.753639
    Duration: 0.006 ms
     Changes:

What else can I do to troubleshoot it so you guys can help me to make it run properly?

**Comment by dougburks** [comment]
Have you tried connecting to the Security Onion web interface from another machine?

Does the machine have Internet access?

Are you able to duplicate the issue on a fresh installation?
